pom.xml

```diff
@@ -263,6 +263,13 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- icu -->
+        <dependency>
+            <groupId>com.ibm.icu</groupId>
+            <artifactId>icu4j</artifactId>
+            <version>69.1</version>
```

Comment:
Shouldn't this dependency have the `provided` scope?

Reply:
Right! I changed that.